### PR TITLE
[docs] Adding documentation of volume-expander

### PR DESCRIPTION
**Pull request by Muneebstakater**

### CODE REVIEW

**Comment by stakater-user** [comment]
@Muneebstakater Image is available for testing. `docker pull docker-stakater-nexus.apps.prod.clusters.stakater.io/stakater/StakaterCloudDocs:SNAPSHOT-PR-263-1fccdb90`

**Comment by stakater-user** [comment]
@Muneebstakater Image is available for testing. `docker pull docker-stakater-nexus.apps.prod.clusters.stakater.io/stakater/StakaterCloudDocs:SNAPSHOT-PR-263-abed48ba`

File: content/sre/storage/volume-expansion.md [hunk 1]
```diff
@@ -0,0 +1,23 @@
+The purpose of the volume-expander is to expand volumes when they are running out of space. This resolves the problem of persistent volumes troubleshooting on passing the set threshold. Volume expander periodically checks the kubelet_volume_stats_used_bytes and kubelet_volume_stats_capacity_bytes published by the kubelets to decide when to expand a volume.  Notice that these metrics are generated only when a volume is mounted to a pod. Also the kubelet takes a minute or two to start generating accurate values for these metrics.
```
Comment (TehreemNisa) [inline content/sre/storage/volume-expansion.md]: The purpose of the volume-expander is to expand volumes (change volumes to persistent volumes)

Comment (TehreemNisa) [inline content/sre/storage/volume-expansion.md]: remove Notice that in second last sentence

File: content/sre/storage/volume-expansion.md [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+    volume-expander-operator.redhat-cop.io/autoexpand: 'true'
+    volume-expander-operator.redhat-cop.io/expand-threshold-percent: "85"
+    volume-expander-operator.redhat-cop.io/expand-by-percent: "20"
+    volume-expander-operator.redhat-cop.io/polling-frequency: "10m"
+    volume-expander-operator.redhat-cop.io/expand-up-to: "1Ti"
+
+When the pvc that is configured with the above annotations is mounted on a pod then volume expander watch the PVC because of the annotation volume-expander-operator.redhat-cop.io/autoexpand annotation and will expand PVC by 20 percent relevant to the current size on reaching the threshold of 85 percent. Volume expander poll the volume metrics after every 10 minutes and then on the bases of kubelet_volume_stats_used_bytes and kubelet_volume_stats_capacity_bytes takes volume expansion decision. When mounted PVC expanded to 1TB then volume expander will not expand PVC further with the error "no space left on device" due to the annotation configured on PVC "volume-expander-operator.redhat-cop.io/expand-up-to: "1Ti" which means that upper bound for this volume is 1TB.
```
Comment (TehreemNisa) [inline content/sre/storage/volume-expansion.md]: add code indentation on annotation and variables

**Comment by stakater-user** [comment]
@Muneebstakater Image is available for testing. `docker pull docker-stakater-nexus.apps.prod.clusters.stakater.io/stakater/StakaterCloudDocs:SNAPSHOT-PR-263-f28df06f`

**Comment by stakater-user** [comment]
@Muneebstakater Image is available for testing. `docker pull docker-stakater-nexus.apps.prod.clusters.stakater.io/stakater/StakaterCloudDocs:SNAPSHOT-PR-263-7e02b93b`

**Comment by stakater-user** [comment]
@Muneebstakater Image is available for testing. `docker pull docker-stakater-nexus.apps.prod.clusters.stakater.io/stakater/StakaterCloudDocs:SNAPSHOT-PR-263-b2dcc6aa`

**Review by usamaahmadkhan — APPROVED** [review]

**Comment by stakater-user** [comment]
@usamaahmadkhan Image is available for testing. `docker pull docker-stakater-nexus.apps.prod.clusters.stakater.io/stakater/StakaterCloudDocs:v0.0.199`

**Pull request merged**